source/tubtrunk/lib/Views/timerView.dart

```diff
@@ -1,14 +1,14 @@
 import 'package:flutter/material.dart';
 import 'package:tubtrunk/Controllers/timerController.dart';
-// import 'package:tubtrunk/Models/RewardMissionModel.dart';
-// import 'package:tubtrunk/Views/MissionView.dart';
-import '../Controllers/NotificationsController.dart';
-import './NotificationView.dart';
 import 'package:circular_countdown_timer/circular_countdown_timer.dart';
 import 'package:flutter_duration_picker/flutter_duration_picker.dart';
 
+import '../Controllers/NotificationsController.dart';
+import './NotificationView.dart';
```

Comment:
Those should be camelCase to match the new filenames

Reply:
done!